src/App.js

```diff
@@ -35,6 +45,7 @@ const removeTodo = (id) => {
 }
   return (
     <>
+    {isLoading?<p>...isloading</p>:<p></p>}
       <h1>Todo List</h1>
       <AddTodoForm onAddTodo={addTodo}/>
       <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>
```

Comment:
Great thank you! 

One more small change here that you can just do in your 1.8 branch since this is already merged: 


```suggestion
      <h1>Todo List</h1>
      <AddTodoForm onAddTodo={addTodo}/>
       {isLoading?<p>...isloading</p>: <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>}
```

We want the loading message to specifically replace the `TodoList`. Right now it has a pretty similar appearance, but eventually you may want to add a special "zero state" to the `TodoList` which will show when you have zero messages. Like this one from my GitHub notifications for example: 

<img width="400" alt="Screen Shot 2023-08-11 at 2 18 22 PM" src="https://github.com/AnastasiaKey/react-todo/assets/12771651/3cd33717-b005-4eb2-b553-8a48daf76d62">

In a situation like that, it'll be important to make sure the `TodoList` can't render until we've gotten the data back from the server and made sure that we actually have 0 messages, not just that we have some but they're not loaded yet.